enterprise/views.py

```diff
@@ -636,27 +636,27 @@ def _enroll_leaner_in_course(
                         )
                     )
                     raise
-                try:
-                    self.create_enterprise_course_enrollment(request, enterprise_customer, course_id, license_uuid)
-                except IntegrityError:
-                    error_code = 'ENTGDS009'
-                    log_message = (
-                        '[Enterprise DSC API] IntegrityError while creating EnterpriseCourseEnrollment.'
-                        'Course: {course_id}, '
-                        'Program: {program_uuid}, '
-                        'EnterpriseCustomer: {enterprise_customer_uuid}, '
-                        'User: {user_id}, '
-                        'License UUID: {license_uuid}, '
-                        'ErrorCode: {error_code}'.format(
-                            course_id=course_id,
-                            program_uuid=program_uuid,
-                            enterprise_customer_uuid=enterprise_customer.uuid,
-                            user_id=request.user.id,
-                            license_uuid=license_uuid,
-                            error_code=error_code,
-                        )
+            try:
+                self.create_enterprise_course_enrollment(request, enterprise_customer, course_id, license_uuid)
+            except IntegrityError:
+                error_code = 'ENTGDS009'
+                log_message = (
+                    '[Enterprise DSC API] IntegrityError while creating EnterpriseCourseEnrollment.'
+                    'Course: {course_id}, '
+                    'Program: {program_uuid}, '
+                    'EnterpriseCustomer: {enterprise_customer_uuid}, '
+                    'User: {user_id}, '
+                    'License UUID: {license_uuid}, '
+                    'ErrorCode: {error_code}'.format(
+                        course_id=course_id,
+                        program_uuid=program_uuid,
+                        enterprise_customer_uuid=enterprise_customer.uuid,
+                        user_id=request.user.id,
+                        license_uuid=license_uuid,
+                        error_code=error_code,
```

Comment:
Does this mean the licensed enrollments are the _only_ enterprise enrollments getting created via this DSC POST view?

Reply:
Yeah currently. That chunk of code that I'm de-denting wasn't supposed to have gotten indented when I put out the check for the license_uuid. According to another conversation with Jeff:

`After talking to Brittney about this it isn't an urgent thing since, as far as we can tell, all enterprise enrollments are either with a license_uuid or defer_creation. Meaning, highlighted code doesn't even get hit anymore 😂 .`

Reply:
OK. This whole thing feels really convoluted to me, but that's not your fault, so I'm happy to go along with this for now.